src/PointwiseFunctions/AnalyticData/GrMhd/MagnetizedFmDisk.cpp

```diff
@@ -115,6 +115,9 @@ MagnetizedFmDisk::MagnetizedFmDisk(
            inverse_plasma_beta_ / b_squared_max);
 }
 
+MagnetizedFmDisk::MagnetizedFmDisk(CkMigrateMessage* msg)
+    : FishboneMoncriefDisk(msg) {}
```

Comment:
Is it enough to call into the `FishboneMoncriefDisk` pup and migration constructor, or should the virtual `evolution::initial_data::InitialData` class also appear here? Or would the two calls into the virtual class conflict somehow?

Reply:
Because the base classes are virtual there is only one base. So the code as-is is correct.